guides/welfare/parental_leave.md

```diff
@@ -37,7 +37,7 @@ The breakdown of pay is as follows if you qualify for SPL:
 
 - 12 weeks at 100% pay (from Made Tech)
 - 6 weeks at 90% pay (statutory shared parental pay)
-- 33 weeks at £151.20 or 90% pay whichever is lower (statutory shared parental pay)
+- 33 weeks at statutory or 90% pay whichever is lower (statutory maternity/adoption pay)
```

Comment:
This looks like a typo, I think it should still say "statutory shared parental pay".